contributing/documentation/format.rst

```diff
@@ -118,7 +118,6 @@ Markup Format        Use It to Display
 ``html+twig``        Twig markup blended with HTML
 ``html+php``         PHP code blended with HTML
 ``ini``              INI
-``php-annotations``  PHP Annotations
```

Comment:
I think, we will have to keep this line as long as we still have examples with annotation syntax.